src/components/GameGrid/GameBoard.tsx

```diff
@@ -32,11 +37,46 @@ const GameBoard: React.FC<Props> = ({ gameData }: Props) => {
       <TextContainer>
         <Shout>Other games</Shout>
 
-        <Loud>Knockout rounds</Loud>
+        <Loud>Round of 32</Loud>
         <Paragraph>Knockout rounds begin on Friday 15 January and end on Saturday 30&nbsp;January.</Paragraph>
       </TextContainer>
 
       <KnockoutRounds knockoutRoundData={gameData.knockout} />
+
+      <TextContainer>
+        <Loud>Round of 16</Loud>
+        <Paragraph></Paragraph>
+      </TextContainer>
+
+      <GroupStages groupStageData={gameData.groupStages} />
```

Comment:
It look as though this component is the same as KnockoutRound.  I'd consider using only one component to avoid code duplication - we can fix this after though (but I do get the customisability of separate components is nice).

Same can be said for Final and SemiFinal